Library/Homebrew/test/test_inreplace.rb

```diff
@@ -85,7 +85,7 @@ def test_sub_gsub
     s.sub!("f", "b")
     assert_equal "boo", s
 
-    s.gsub!("o", "e")
+    s.gsub!("o", "e") # rubocop:disable Performance/StringReplacement
```

Comment:
Because under current context we are testing the method of `String#gsub!`, while rubocop will try to replace it to `tr!`.